### PR TITLE
The Great Radio Rework: NTNET Part 1 of many.

### PR DESCRIPTION
**Pull request by Dejaku51**
## About The Pull Request
port of https://github.com/tgstation/tgstation/pull/54462 so i can port NTNET wiremod components etc

### CODE REVIEW

**Comment by francinum** [comment]
I'll have a look at this but this is liable to be incredibly counterproductive to an aim I've had of *gutting NTNet completely*

**Comment by Dejaku51** [comment]
waiting for review

**Comment by francinum** [comment]
It's more that it's far more complicated than native radio for no justifiable reason.

**Comment by Dejaku51** [comment]
does that mean I'll have to wait a long time before this will be merged?

**Comment by francinum** [comment]
Fuck it, sure, whatever.
I don't have the bandwidth to give a fuck about cleaning up after TG designing themselves into a corner and not using their own tools to claw their way out.

File: code/controllers/subsystem/processing/networks.dm
Comment (Archanial) [inline code/controllers/subsystem/processing/networks.dm]: this should check for virtual z levels

File: code/modules/NTNet/netdata.dm
Comment (Archanial) [inline code/modules/NTNet/netdata.dm:32]: 😆

**Review by Archanial — DISMISSED** [review]
Looks fine, still needs testmerge

**Comment by Dejaku51** [comment]
Oh fuck off

**Pull request closed without merging**
